### DISCUSSION TITLE
Any update on a release (candi)date?

### DISCUSSION THREAD

**Discussion by bitfactory-frank-spee** [shopware/frontends#145] [General]
I can't find any update on a release (candi)date of Shopware Frontends (which was promised Q1 2023). Is there any solid news?

When SWF is not ready yet I think we are stuck with Shopware PWA to build a production version of a shopware headless shop. But Shopware PWA is running out of time. Node.js 16 is EOL on September the 11th 2023.

Will Shopware Frontends be released before that, or is there any update on Node.js 18 / Nuxt 2.16.x support for Shopware PWA?

Has anyone already tried to use the current version of Shopware Frontends and can express how stable it is? Is it already doable to create a new shop with it ready for production in Q3/Q4 2023?

It is very unclear to me what our options are with Shopware PWA / Shopware Frontends, it looks like choosing either will give me major headaches (read: alot of extra unneeded hours / (security) risks) in Q3/Q4 later this year.

**Comment by BrocksiNet** [comment]
_We have discussed this in the community Slack. You can find the discussion [here](https://shopwarecommunity.slack.com/archives/C050L6NCMGQ/p1681735663365419). I'm just summarizing this here for others._

**What is currently in progress:**

- There will be a [landing page on shopware.com](https://www.shopware.com/en/solutions/shopware-composable-frontends/) about frontends :heavy_check_mark: 
- There will be a new, automatically generated [API client](https://www.npmjs.com/package/@shopware/api-client) with autosuggest for parameters and API routes (depending on instance)  :heavy_check_mark: 
- We need to integrate the new API client into the composables and test the complete application :construction: 
- We are working on simplifying multi-store setups and translations in general :construction: 
- We are also working on migrating the complete project from gitlab to github  :heavy_check_mark: 
- We are already working on the issues you provide via github (thanks for this input) :heavy_check_mark:

**Will there be a release (candi)date?**

Shopware composable frontends is like a toolbox. So we provide packages that you can use wherever you want. The demo store is like a showcase or playground, not a blueprint for your project. You can build on the same stack as we did for the demo store and reuse some or all of the composables, depending on your needs. But you are always free to write your own composables, include other services, or extend existing composables for your needs. In the end, there is not really a released bundle because we don't know what packages you are using or in what environment, e.g. Astro or React or Vue.

But still we discuss to celebrate a release more like a marketing release for example when the new API client is ready to celebrate it with version v1.0.0. In general Shopware composable frontends is already stable because it only depends on the store API so if there are no breaking changes all composables will working after updates and so on.

**Comment by bitfactory-frank-spee** [comment]
👍 So there is now a new API client, see also https://shopwarecommunity.slack.com/archives/C050L6NCMGQ/p1687964264563839

**Reply by BrocksiNet** [reply]
And there is also a nice bot for the next releases that are creating new PRs. :smile: 

Look here: https://github.com/shopware/frontends/pull/293

:rocket:

**Reply by patzick** [reply]
Updates: https://github.com/shopware/frontends/discussions/471 took a bit longer than we expected, but we hope the results will be worth the wait :)
